scim/data_service_principal.go

```diff
@@ -0,0 +1,64 @@
+package scim
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// DataSourceServicePrincipal returns information about the spn specified by the application_id
+func DataSourceServicePrincipal() *schema.Resource {
+	type spnData struct {
```

Comment:
Can we reuse the entity from SPN resource, by the way?

Reply:
this returns a few more computed fields